locales/leatherman.pot

```diff
@@ -6,7 +6,7 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: leatherman 0.3.6\n"
+"Project-Id-Version: leatherman 0.3.7\n"
```

Comment:
Actually, a build may not catch this because it depends on source files changing. Might have to fix it up to update if the version changes. It has no functional impact that I'm aware of however, so we could also maybe just remove version from the template.